website/docusaurus.config.js

```diff
@@ -45,6 +45,15 @@ const config = {
         },
       }),
     ],
+    [
+      '@docusaurus/preset-classic',
+      {
+        googleAnalytics: {
+          trackingID: 'G-RX9N8G7RS5',
```

Comment:
can you add the maintainer to this?

Reply:
I have added you. For other maintainers please let me know your id.

Reply:
@maxsmythe Yes. I created a GA account where I can add users to grant access. Please let me know if there is any better way to create and manage GA accounts for both GK websites.

Reply:
Don't know much about analytics sadly :/

What kind of ID did you need?

Reply:
@maxsmythe Your Gmail id. I can grant you access to that.

Reply:
Oh, perfect! smythe@google.com , LMK if that doesn't work